Makefile

```diff
@@ -248,7 +247,7 @@ packaging/centos-systemd/.uptodate: $(wildcard packaging/centos-systemd/*)
 	touch $@
 
 ifeq ($(BUILD_IN_CONTAINER), true)
-dist-packages: dist/agent-linux-amd64 dist/agentctl-linux-amd64 build-image/.uptodate
+dist-packages: dist-agent dist-agentctl build-image/.uptodate
```

Comment:
Depending on the general target makes this much shorter than it would be otherwise